packages/terra-section-header/tests/jest/SectionHeader.test.jsx

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import SectionHeader from '../../src/SectionHeader';
+
+describe('SectionHeader', () => {
+  const defaultRender = <SectionHeader title="foo" />;
+
+  // Snapshot Tests
+  it('should render a default component', () => {
+    const wrapper = shallow(defaultRender);
+    expect(wrapper).toMatchSnapshot();
+  });
+
+  it('should render without an accordion icon when no \'onClick()\' is passed', () => {
+    const sectionHeader = <SectionHeader title="foo" />;
+    const wrapper = render(sectionHeader);
+    expect(wrapper).toMatchSnapshot();
+  });
+
+  it('should render with an accordion icon when an \'onClick()\' callback is passed', () => {
+    const sectionHeader = <SectionHeader title="foo" onClick={() => {}} />;
+    const wrapper = render(sectionHeader);
+    expect(wrapper).toMatchSnapshot();
+  });
+
+  it('should render with an accordion icon in the open position when an \'onClick()\' callback and \'isOpen\' is passed', () => {
+    const sectionHeader = <SectionHeader title="foo" onClick={() => {}} isOpen />;
+    const wrapper = render(sectionHeader);
+    expect(wrapper).toMatchSnapshot();
+  });
+
+  // Structure Tests
+  it('should have the class section-header', () => {
+    const wrapper = shallow(defaultRender);
```

Comment:
Feels like the original snapshot would be testing this.

Reply:
Right here. https://github.com/cerner/terra-core/pull/1313/files#diff-59ded93eb2bcb2fa58e32334c3553d49R5. Given that, we don't need this test.

Reply:
Generated by the module-generator, removed 1d52efb159e8954344ed0895aecf6ce34cd356b8